.github/workflows/build.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Build
+name: Update and build application
```

Comment:
Shouldn't the name be more of a title? Like "Update and Build Application"

Reply:
It's a sentence, not a blog title

Reply:
It's a sentence, not a blog title imo but your personal flavor may differ